api/src/main/resources/db/migration/V2023.02.08__collection_add_description.sql

```diff
@@ -0,0 +1,5 @@
+USE osmt_db;
+
+ALTER TABLE `Collection`
+    ADD COLUMN `description` text

```

Comment:
In many relational DBs, a varchar column is stored in the clustered index, right on the page being read for PK access. A text column is a pointer to a separate blob, and by default requires an additional read. While this is a completely valid operation, for a textual value that has reasonable boundaries, it is much more efficient to express as char or varchar.

Reply:
I was referencing `statement` field of `RichSkillDescriptor` it used a `text` column, so I used the same for uniformity.

https://github.com/wgu-opensource/osmt/blob/develop/api/src/main/resources/db/migration/V2020.09.15__initial_schema.sql#L111

It can be changed if you'd like. However, I remember talking about length limit in the meeting a few weeks agon and I think it was said that we didn't want one. If we switched to `varchar` what would we want for a character limit?

Reply:
`JobCode.description` is a `text` column as well.

https://github.com/wgu-opensource/osmt/blob/develop/api/src/main/resources/db/migration/V2020.09.15__initial_schema.sql#L52

Reply:
We've got a lot of those in the schema, and a lot of inconsistency in naming and technical conventions. Let's not block this work for it now. Thanks for the feedback.